src/components/core/DpDataTable/DpTableHeader.vue

```diff
@@ -1,13 +1,73 @@
+<template>
+  <tr
+    ref="tableHeader"
+    :class="{ 'c-data-table__sticky-header': isSticky }">
+    <th
+      v-for="(hf, idx) in headerFields"
+      scope="col"
+      :data-col-idx="`${idx}`">
+      <ResizableColumns
+        v-if="isResizable"
+        :is-last="headerFields.length === idx"
+        :header-field="hf"
+        :idx="idx">
```

Comment:
does v-text works here too?

Reply:
v-text doesnt work in combination with slots. thanks for the question to think about and to try it